File: solutions/0867. Transpose Matrix/0867.java
class Solution {
  public int[][] transpose(int[][] A) {
    int[][] ans = new int[A[0].length][A.length];

    for (int i = 0; i < A.length; ++i)
      for (int j = 0; j < A[0].length; ++j)
        ans[j][i] = A[i][j];

    return ans;
  }
}

File: solutions/0867. Transpose Matrix/0867.cpp
class Solution {
 public:
  vector<vector<int>> transpose(vector<vector<int>>& A) {
    vector<vector<int>> ans(A[0].size(), vector<int>(A.size()));

    for (int i = 0; i < A.size(); ++i)
      for (int j = 0; j < A[0].size(); ++j)
        ans[j][i] = A[i][j];

    return ans;
  }
};
